src/types/index.ts

```diff
@@ -165,6 +165,13 @@ export interface SelectProps {
      * @default "Select..."
      */
     placeholderText?: string;
+    /**
```

Comment:
Don't you need generate typedoc again?

Reply:
No, I don't want because I want to generate new docs after release new version to npm